docs/source/support/release-notes.rst

```diff
@@ -6,6 +6,9 @@ Release notes
 
 This page contains release notes for the SDK.
 
+
+- Made the archive CLI arguments optional. See `#1905 <https://github.com/digital-asset/daml/issues/1905>`__.
```

Comment:
New release notes should go to `unreleased.rst`, also note the changed release notes format.

Reply:
oh, right. will change that.

Reply:
it's done